db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20171010220806) do
+ActiveRecord::Schema.define(:version => 20171105200238) do
```

Comment:
I think the changes to this file should reflect that the portal_semesters table was dropped. However I haven't dropped that many tables so perhaps the schema doesn't get updated the right way?

Reply:
Not sure what happened here. I've re-run the migration and it removes this table from the schema.rb. I've pushed that change back to this branch.